src/app/beer_garden/api/http/schemas/v1/command_publishing_blocklist.py

```diff
@@ -0,0 +1,9 @@
+from marshmallow import Schema, fields
+
+
+class CommandPublishingBlocklistSchema(Schema):
```

Comment:
As referred to in earlier you'll want a CommandPublishingBlocklistListSchema (as silly as that name sounds) that facilitates returning a list of things.  Again, see the UserListSchema as a reference.